go.mod

```diff
@@ -9,14 +9,15 @@ require (
 	github.com/prometheus/client_golang v1.12.1
 	github.com/satori/go.uuid v1.2.0
 	github.com/smartcontractkit/chainlink v1.3.0
-	github.com/smartcontractkit/chainlink-relay v0.0.0-20220422131434-0cef40354ecc
+	github.com/smartcontractkit/chainlink-relay v0.0.0-20220514135708-91ac8dd2d069
 	github.com/smartcontractkit/libocr v0.0.0-20220414173908-cdfa6bef133a
 	github.com/smartcontractkit/terra.go v1.0.3-0.20220108002221-62b39252ee16
 	github.com/stretchr/testify v1.7.1
 	github.com/tendermint/tendermint v0.34.15
 	github.com/terra-money/core v0.5.14
 	go.uber.org/multierr v1.8.0
 	go.uber.org/ratelimit v0.2.0
+	go.uber.org/zap v1.21.0
```

Comment:
It would be cool is this wasn't a direct dependency. It's only needed for the log levels. I don't feel strongly about it, but you could alias the levels in `pkg/logger` to prevent this. Wdyt?

Reply:
I *think* some imports would still leak through because of the observed logs...maybe not... :thinking: